fastlane_core/spec/fastlane_folder_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe FastlaneCore::FastlaneFolder do
+  let (:fastfile_name) { "Fastfile" }
+  describe "#path" do
+    it "returns the fastlane path if it exists" do
+      expect(FastlaneCore::FastlaneFolder.path).to eq("./fastlane/")
+    end
+
+    it "returns nil if the fastlane path isn't available" do
+      Dir.chdir("/") do
+        expect(FastlaneCore::FastlaneFolder.path).to eq(nil)
+      end
+    end
+
+    it "returns the path if fastlane is being executed from the fastlane directory" do
+      Dir.chdir("fastlane") do
+        expect(FastlaneCore::FastlaneFolder.path).to eq('./')
+      end
+    end
+  end
+
+  describe "#fastfile_path" do
```

Comment:
`#` indicates instance method in ruby, so this is confusing at least to me.
If we have some guidelines for this, it would be ok 👍 

### [In ruby documentation](http://ruby-doc.org/documentation-guidelines.html)

- `.` (or `::`) for class method
- `#` for instance method

e.x. [`class File`](https://docs.ruby-lang.org/en/2.4.0/File.html)